src/components/FeatureViews/TeachingLog/TeachingLogContainer.js

```diff
@@ -279,7 +279,10 @@ export default function TeachingLogContainer() {
 									<b>Total Hours</b>
 								</TableCell>
 								<TableCell>
-									{sessions.length > 0 && summaryLog.reduce((acc, course) => (acc.hours + course.hours))}
+									{sessions.length > 0 && 
+									summaryLog.reduce((acc, course) => (acc.hours + course.hours)) === Number.NaN 
```

Comment:
Since this error isn't too descriptive anyways, you could probably make this more readable and DRY by using optional chaining. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
`{sessions.length > 0 ? summaryLog.reduce((acc, course) => (acc?.hours + course?.hours)) : "Error"`
Just to check for understanding, if the hours didn't exist in acc or course, that reduce would return undefined due to optional chaining instead of NaN, and thus would still display "Error", correct?